config/manifests/bases/osp-director-operator.clusterserviceversion.yaml

```diff
@@ -4,9 +4,9 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.2.0
-    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+    operators.operatorframework.io/builder: operator-sdk-v1.1.0
```

Comment:
I don't think there should be changes to this file either right?

Reply:
Ack